Podfile

```diff
@@ -10,6 +10,7 @@ target 'ios-base' do
   pod 'Fabric', '~> 1.9.0'
   pod 'Crashlytics', '~> 3.12.0'
   pod 'R.swift', '~> 5.0.3'
+  pod 'SwiftLint'
```

Comment:
Follow the same notation and after running the `pod install` command, specify the current version `, '~> x.x.x'`